scripts/globals/spells/trust/yoran-oran_uc.lua

```diff
@@ -16,6 +17,31 @@ end
 spellObject.onMobSpawn = function(mob)
     xi.trust.message(mob, xi.trust.messageOffset.SPAWN)
 
+    -- TODO: Nott weaponskill needs implemented and logic added here for Yoran-Oran to use at 50% MP at level 50.
+    -- TODO: UC trusts are supposed to get bonuses depending on unity ranking. Needs research.
+    -- TODO: Custom spawn messages if Unity ranking is higher.
+    -- TODO: Figure out what level of Fast Cast he has.
+
+    mob:addMod(xi.mod.CURE_POTENCY, 50)
+    mob:addMod(xi.mod.REGAIN, 50)
+    mob:addMod(xi.mod.MPP, 15) -- This is supposed to increase with Unity rank, but I don't believe that's implemented so it is set to the minimum.
+    mob:addSimpleGambit(ai.t.SELF, ai.c.NOT_STATUS, xi.effect.AFFLATUS_SOLACE, ai.r.JA, ai.s.SPECIFIC, xi.ja.AFFLATUS_SOLACE)
+    mob:addSimpleGambit(ai.t.PARTY, ai.c.HPP_LT, 25, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.CURE)
+    mob:addSimpleGambit(ai.t.PARTY, ai.c.HPP_LT, 75, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.CURE)
+    mob:addSimpleGambit(ai.t.PARTY, ai.c.NOT_STATUS, xi.effect.PROTECT, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.PROTECTRA)
+    mob:addSimpleGambit(ai.t.PARTY, ai.c.NOT_STATUS, xi.effect.SHELL, ai.r.MA, ai.s.HIGHEST, xi.magic.spellFamily.SHELLRA)
```

Comment:
There -ra's probably want to be PARTY_MULTI(https://github.com/LandSandBoat/server/blob/base/scripts/globals/gambits.lua#L23)

Reply:
Done.

Reply:
Hmm.... he's stopped casting Protectra/Shellra under the PARTY_MULTI condition. Should I revert it or is there something else going on?

Reply:
You can check the conditions and debug/see:
https://github.com/LandSandBoat/server/blob/base/src/map/ai/helpers/gambits_container.cpp#L229

Presumably the predicate that picks a target to cast on isn't actually doing that, or it isn't handled correctly:
https://github.com/LandSandBoat/server/blob/base/src/map/ai/helpers/gambits_container.cpp#L290

For the sake of getting this PR in, just set it back to PARTY

Reply:
Done. I'll look at it in my own time.